docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+services:
+  wg-access-server:
+    # to build the docker image from the source
+    # build:
+    #   dockerfile: Dockerfile
+    #   context: .
+    image: place1/wg-access-server
```

Comment:
Thanks for adding this. I intend to create a documentation site soon and I'll include docker-compose as an option for getting started.